pillar.example

```diff
@@ -2,14 +2,38 @@ netweaver:
   virtual_addresses:
     192.168.201.111: hacert01
     192.168.201.112: hacert02
+    192.168.201.113: hacert03
     192.168.201.115: sapha1as
     192.168.201.116: sapha1er
-  # Clean /sapmnt/{sid} and /usr/sap/{sid}/SYS content. True by default
-  clean_nfs: true
+    192.168.201.117: sapha1pas
+  # Clean /sapmnt/{sid} and /usr/sap/{sid}/SYS content. It will only work if ASCS node is defined.
+  # True by default
+  clean_nfs: True
   swpm_media: your_nfs_share_with_swpm_and_sapexe_folders
   sapmnt_inst_media: your_nfs_share_SID_folder
   swpm_folder: your_swpm_folder_relative_to_swpm_media
   sapexe_folder: your_sapexe_sar_folder_relative_to_swpm_media
+  # Folder where the installation files are stored. /tmp/unattended by default. Set None to use
+  # SAP default folders (it will only work with ASCS and ERS)
+  installation_folder: /tmp/unattended
+  # DB/PAS/AAS instances require additional DVD folders like NW Export or HDB Client folder
```

Comment:
Hi @arbulu89 do these folders contain some installation files/ or any other files we need to provide? I did not understand whats the purpose of these DVDs